### PR TITLE
Add appContextMap entry for CIT context in createToken.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #27.

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
(Adding the context map to decoding, too.)

File: lib/tokens.js [hunk 1]
```diff
@@ -412,7 +415,9 @@ async function _createToken({
     meta: `z${base58.encode(attributes)}`,
     payload: `z${base58.encode(payload)}`,
   };
-  const token = await cborldEncode({jsonldDocument, documentLoader});
+
```
Comment (dlongley) [inline lib/tokens.js:418]:
```suggestion
```

**Comment by dlongley** [comment]
Note that this PR doesn't update to the latest cborld that has the document loader fixes. I don't think that affects output in anyway, but we should do that as soon as we can.

**Comment by dmitrizagidulin** [comment]
@dlongley oh, good point. I updated to the latest `cborld` version, with the documentLoader fixes.

**Comment by dmitrizagidulin** [comment]
Downgraded back to cborld 1.0 after conversation with manu and dave; upgrade will need to be sorted out separately.

**Pull request merged**